src/test/java/org/sonatype/nexus/ci/config/ComToOrgMigratorIntegrationTest.groovy

```diff
@@ -91,21 +93,12 @@ class ComToOrgMigratorIntegrationTest
 
   def 'it migrates a Freestyle IQ job'() {
     when:
+      GroovyMock(IqUtil, global: true)
```

Comment:
If you get rid of the IqUtil.verifyOrCreateApplication then you won't have to do this.

Reply:
> If you get rid of the static method IqUtil.verifyOrCreateApplication then you won't have to do this if you just call directly from IqClient

Changed df803ed